packages/expo-blur/android/src/main/java/expo/modules/blur/ExpoBlurView.kt

```diff
@@ -3,57 +3,83 @@ package expo.modules.blur
 import android.annotation.SuppressLint
 import android.content.Context
 import android.graphics.Color
-import android.os.Build
-import android.view.ViewGroup
 import eightbitlab.com.blurview.BlurView
-import eightbitlab.com.blurview.RenderEffectBlur
 import expo.modules.blur.enums.BlurMethod
 import expo.modules.blur.enums.TintStyle
 import expo.modules.kotlin.AppContext
-import expo.modules.kotlin.exception.Exceptions
 import expo.modules.kotlin.views.ExpoView
 
+private enum class BlurViewConfiguration {
+  UNCONFIGURED,
+  NONE,
```

Comment:
What's the difference between `UNCONFIGURED` vs `NONE`? It would be nice to add maybe a comment on why we need those two.